### ISSUE TITLE
Sophia out of action.  Bed stuck.  

### ISSUE THREAD

**Issue by dazz-ling** [DoESLiverpool/somebody-should#968]
There are a few issues with Sophia today. 
The focus was widely out plus the bed has been jammed really high ( about a cm higher than a flat bed) and seems to have gotten stuck. 
From memory,   you have to release the bed from underneath the base,  so a 2 person job. 
Nobody has it booked for the rest of the week.  
Darran

**Comment by amcewen** [comment]
I've booked Sophia's calendar out for the rest of this week and all of next (and marked it as "busy") so nobody else will be able to book her, and added the `Broken` label here so she gets marked as out-of-action on status.doesliverpool.com.

**Comment by magman2112** [comment]
Can we possibly find out who was last using Sophia, as it looks like we may need to do a bit of re-training so that we don’t get this issue again with this user?

**Comment by DoESsean** [comment]
Looks like Manmohan was the last booking on Sophia, although he has been in
recently too and no problems were reported after his last visit, so it
would be strange if he needed completely retraining.

**Comment by magman2112** [comment]
I didn’t say complete retraining, just a bit of retraining. I would also like to know what they were trying to do which caused the bed to go out of alignment, so that we can potentially add it as an item in the laser induction, to reduce the the incidents in the future.

**Comment by MatthewCroughan** [comment]
Only just seeing this, they had left the bed on Sophia really far down, and I was unsure how the ruler worked. Adjusted it too tightly up and didn't realise there was an upper limit

I think we need a bit of editing in the wiki, it doesn't describe how to use the ruler, I wad never properly inducted on sophia in particular. The current wiki statement is "align lens with bottom of ruler", not specifying what the bottom is, and whatever I thought the bottom was caused me to tighten the bed too much.

Will make my own issue immediately after these discoveries in the future, but was incredibly exhausted at 5:30am when it occured.

**Comment by magman2112** [comment]
OK, so the retraining is for Matt then :-). We can update the wiki, as Matt has suggested, once Sophia is back up and working. We might even get around to making a new gauge for Sophia as well, possibly with the instructions engraved on it.

**Comment by johnmckerrell** [comment]
Thanks Matt for owning up and giving a good explanation for why and tips for improvements. Please consider not using the equipment when very tired though!

**Comment by magman2112** [comment]
Sophia is back up and working again. I as able to release the bed and get access to the belt tensioner for the bed raising screws. With the belt tension released it was relatively easy to reset the bed to level.

With help from @Sean-anotherone, we then had a great deal of fun getting the laser aligned again. The laser alignment is a real pain on Sophia, so we have modified one of the mirror mounts to see if we can improve the alignment process. We will wait to see if this modification is stable enough before modifying any of the other mirror mounts.

I also gave Sophia a good clean out whilst the laser bed was loose.
